test/uk/gov/hmrc/tai/service/benefits/CompanyCarServiceSpec.scala

```diff
@@ -149,6 +149,14 @@ class CompanyCarServiceSpec extends PlaySpec
         val expectedResult = TaiNoCompanyCarFoundResponse("No company car found")
         Await.result(sut.beginJourney(generateNino, 1), 5 seconds) mustBe expectedResult
       }
+      "there are no employments" in  {
```

Comment:
This new test is verifying that a runtime exception is thrown, so I think it needs to sit outside of the current test block which is grouping all situations that return a TaiNoCompanyCarFoundResponse